ai_summarizer/admin.py

```
@@ -1,9 +1,14 @@
 from .models import AISummary
 
+
 @admin.register(AISummary)
 class AISummaryAdmin(admin.ModelAdmin):
-    list_display = ('patient', 'resource_type', 'created_at')
-    list_filter = ('resource_type', 'created_at')
-    search_fields = ('patient__full_name', 'resource_type', 'summary')
-    readonly_fields = ('id', 'created_at')
\ No newline at end of file
+ class AISummaryAdmin(admin.ModelAdmin):
+-    list_display = ('patient', 'resource_type', 'created_at')
```

Ruff (0.12.2)

SyntaxError: Expected an indented block after `class` definition

---

SyntaxError: Invalid assignment target

---

SyntaxError: Invalid assignment target

---

SyntaxError: Unexpected indentation

---

Line too long (106 > 88)

(E501)